dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -591,7 +591,9 @@
                                 <ColumnDefinition Width="*" />
                             </Grid.ColumnDefinitions>
 
-                            <Viewbox x:Name="IconBox"                                
+                            <Viewbox x:Name="IconBox"
+                                Width="16"
+                                Height="16"
                                 Margin="{ThemeResource NavigationViewItemOnLeftIconBoxMargin}">
```

Comment:
If you unpack these two numbers: IconColumn = 40, and this Margin = 12,0,12,0, you end up with 40-24=16. So the Width should already be accounted for. Do you need both?

Reply:
Right, probably don't need width, will remove.